daily-shoutout/app.py

```diff
@@ -55,11 +56,22 @@ def sendTweet(tweet_string):
     client.create_tweet(text = tweet_string)
 
 
+def returnResponse(response_string):
+    return{
+        "statusCode": 200,
+        "body": json.dumps(response_string)
+    }
+
+
 def lambda_handler(event, context):
     person = people[get_random_index(len(people))]
     cause = causes[get_random_index(len(causes))]
+    message_string = f"Noted billionare {person} once again contributes nothing to {cause}."
+    if event.get("source", False) == "aws.events":
```

Comment:
If you go with the `None` default, I think this would need to become:

```suggestion
    if event and event.get("source") == "aws.events":
```

Reply:
This is a good find. It needs to be mocked as a dict for local development.